blogs/helpers.py

```diff
@@ -1,20 +1,58 @@
+import threading
+import bleach
+from bs4 import BeautifulSoup
+from django.contrib.sites.models import Site
+from requests.exceptions import ConnectionError
+from django.core.mail import send_mail, get_connection, EmailMultiAlternatives
+import mistune
 import requests
-import json
+from django.http import Http404
+import subprocess
 from django.conf import settings
-from markdown import Markdown
-from io import StringIO
-
-def get_root(extracted, subdomain):
-    if extracted.suffix:
-        return "https://{}.{}.{}".format(subdomain, extracted.domain, extracted.suffix)
+from _datetime import timedelta
+import geoip2
+from django.contrib.gis.geoip2 import GeoIP2
+
+
+def root(subdomain=''):
+    domain = ''
+    
+    try:
+        domain = Site.objects.get_current().domain
+    except Exception as e:
+        pass
```

Comment:
wasn't sure what else to do here since it appears to be executing when `python manage.py migrate` is executed and before table exists